### DISCUSSION TITLE
Excerpts from David Dunn's Plicsot for neoscore Formalization

### DISCUSSION THREAD

**Discussion by jgarte** [DigiScore/neoscore#43] [Ideas]
![2022-06-22-191821_1057x841_scrot](https://user-images.githubusercontent.com/47760695/175181651-4d9d8111-d900-4c98-bc00-488fec7c2ade.png)

[Purposeful Listening In Complex States of Time](http://artsdocbox.com/Music/99408664-Purposeful-listening-in-complex-states-of-time.html) (1997-98): 20 compositions for solo listener by David Dunn might serve as a nice experimental music notation example to formalize with `neoscore`.

@ajyoon  WDYT

What `neoscore` primitives could model the notation in the following excerpt?

![2022-06-22-191833_1076x375_scrot](https://user-images.githubusercontent.com/47760695/175181659-caefd346-c39f-4227-b245-b7ff67d60d1e.png)

**Comment by ajyoon** [comment]
This would be a great use-case for neoscore's extensibility. Implementing it would require building up a collection of new score-specific classes which formalize the notation system here and represent it with `core`-module classes. It would be much like the Feldman example in that it would use very little, if any, of the `western` module, instead primarily using simple `Path` and `Text` objects.
